### PR TITLE
Forward `ExternalLink` component ref

### PR DESCRIPTION
**Pull request by owi92**
This is necessary for (hopefully) upcoming changes in appkit that will allow users to navigate header menus with arrow keys.

Addresses part of #753 (other menus like the series block sorting and `add content` menus still need some work)
Related appkit PR: https://github.com/opencast/appkit/pull/3

### CODE REVIEW

File: frontend/src/relay/auth.tsx [hunk 1]
```diff
@@ -58,7 +59,11 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
             link: () => <Link className={className} to={target.toString()} target="_blank">
                 {children}
             </Link>,
-            button: () => <LinkButton className={className} to={target.toString()} target="_blank">
+            button: () => <LinkButton
+                className={className}
+                to={target.toString()}
+                target="_blank"
+            >
```
Comment (LukasKalbertodt) [inline frontend/src/relay/auth.tsx:66]: Nothing changed here, right? Is this change intended?

File: frontend/src/relay/auth.tsx [hunk 2]
```diff
@@ -78,14 +83,15 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
         <input type="hidden" name="jwt" />
 
         <button
+            {...{ ref }}
             className={className}
```
Comment (LukasKalbertodt) [inline frontend/src/relay/auth.tsx:87]: Then you can also combine it with the next line 
```suggestion
            {...{ ref, className }}
```

File: frontend/src/relay/auth.tsx (hunk 1 above)
Comment (owi92) [reply frontend/src/relay/auth.tsx]: Oh right, I used a different indentation first where line 61 would exceed 100 chars. No longer the case though.

**Review by LukasKalbertodt — APPROVED** [review]
